projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/integration-card/integration-card-body.js

```diff
@@ -5,18 +5,26 @@ const IntegrationCardBody = ( { isExpanded, children, cardData = {} } ) => {
 		return null;
 	}
 
-	const { notInstalledMessage, notActivatedMessage, isInstalled, isActive, isLoading } = cardData;
+	const { notInstalledMessage, notActivatedMessage, isInstalled, isActive, isLoading, type } =
+		cardData;
+
+	const isPlugin = type === 'plugin';
+	const isService = type === 'service';
+
+	const showPluginInstallMessage = isPlugin && ! isInstalled;
+	const showPluginActivateMessage = isPlugin && isInstalled && ! isActive;
+	const showContent = ( isPlugin && isInstalled && isActive ) || isService;
```

Comment:
I had to adjust these conditionals to handle services, not just plugins. As the conditionals were looking complex, I just pulled them out and defined them up here to use in the return statement.